lib/grplatt.gi

```diff
@@ -2068,6 +2068,23 @@ local G,	# group
 	    if onlysimple then
 	      # all groups obtained will have a solvable factor
 	      l:=[];
+            elif HasElementaryAbelianFactorGroup(j,N) then
+              #Error("invar");
+              r:=ModuloPcgs(j,N);
+              jg:=RelativeOrders(r)[1];
+              l:=MTX.BasesSubmodules(GModuleByMats(LinearActionLayer(G,r),
+                GF(jg)));
+	      Info(InfoLattice,2,"found ",Length(l)," submodules");
+	      idx:=LogInt(Index(j,M),jg);
```

Comment:
Something is funny about the indentation here; tabs vs. spaces? Also again in lines 2085-2087

Reply:
It probably still is tabs (of which the file has many), while the new line only includes spaces. To fix this I think I would have to replace tabs by spaces overall (which will make the diff look large but otherwise is harmless)?